radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java

```diff
@@ -20,14 +20,14 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.radixdlt.client.core.atoms.particles.Particle;
 import com.radixdlt.client.core.atoms.particles.Spin;
-import com.radixdlt.client.atommodel.storage.StorageParticle;
+import com.radixdlt.client.atommodel.message.MessageParticle;
 import com.radixdlt.client.atommodel.timestamp.TimestampParticle;
 import com.radixdlt.client.atommodel.tokens.OwnedTokensParticle;
 import com.radixdlt.client.core.crypto.ECPublicKey;
 import com.radixdlt.client.core.crypto.ECSignature;
 
 /**
- * An atom is the fundamental atomic unit of storage on the ledger (similar to a block
+ * An atom is the fundamental atomic unit of message on the ledger (similar to a block
```

Comment:
"unit of message"? lol

Reply:
hahaha and I thought I was so careful to revert all those unnecessary changes.. apparently not